sklearn/linear_model/tests/test_ridge.py

```diff
@@ -715,3 +716,57 @@ def test_ridge_fit_intercept_sparse():
     assert_warns(UserWarning, sparse.fit, X_csr, y)
     assert_almost_equal(dense.intercept_, sparse.intercept_)
     assert_array_almost_equal(dense.coef_, sparse.coef_)
+
+
+def make_noisy_forward_data(
+    n_samples=100,
+    n_features=200,
+    n_targets=10,
+    train_frac=.8,
+    noise_levels=None,
+    random_state=42):
```

Comment:
please use a more classical formarting for the args. It's confusing this way.

For instance:

```python
def make_noisy_forward_data(n_samples=100, n_features=200, n_targets=10,
                            train_frac=.8, noise_levels=None, random_state=42):
     ...
```